packages/nbdime/src/common/mergeview.ts

```diff
@@ -409,6 +424,92 @@ const conflictMarkerLineChunkMappingField = StateField.define<
   },
 });
 
+/**
+ * uncollapseEffect effect to synchronize uncollapseEffectd ranges between editors.
+ */
+const uncollapseEffect = StateEffect.define<number>({
+  map: (value, change) => change.mapPos(value),
+});
+
+/**
+ * Effect to set the collapser widgets in an editor
+ */
+const setCollapsersEffect = StateEffect.define<DecorationSet>({
+  map: (value, mapping) => value.map(mapping),
+});
+
+class CollapseWidget extends WidgetType {
+  constructor(
+    readonly lines: number,
+    readonly siblings: { line: number; cm: EditorView }[],
+  ) {
+    super();
+  }
+
+  eq(other: CollapseWidget) {
+    return (
+      this.lines == other.lines &&
+      this.siblings.every(
+        sibling =>
+          other.siblings.findIndex(
+            otherSibling =>
+              otherSibling.line == sibling.line &&
+              otherSibling.cm == sibling.cm,
+          ) >= 0,
+      )
+    );
+  }
+
+  toDOM(view: EditorView) {
+    let outer = document.createElement('div');
+    outer.className = 'cm-collapsedLines';
+    outer.textContent = view.state.phrase('...$ unchanged lines...', this.lines);
```

Comment:
@ vidartf I'm changing for the following:

```suggestion
    outer.textContent = view.state.phrase('(…$ unchanged lines…)', this.lines);
```